Sources/Epoxy/Row/ImageRow.swift

```diff
@@ -0,0 +1,152 @@
+//  RakuyoKit
+//
+//  Created by Rakuyo on 2024/5/17.
+//  Copyright © 2024 RakuyoKit. All rights reserved.
+//
+
+import UIKit
+
+import EpoxyCore
+import RAKCore
+
+// MARK: - ImageRow
+
+/// Replace `UIImageView` in Epoxy component
+///
+/// If you want to extend, consider building your own view with
+/// the help of `ImageRow.Style`, `ImageRow.Content` and `ImageRow.Behaviors`.
+public final class ImageRow: UIImageView { }
+
+// MARK: StyledView
+
+extension ImageRow: StyledView {
+    public struct Style: Hashable {
+        /// The tint color.
+        public let tintColor: UIColor?
+
+        /// The content mode determines the layout of the image when its size does
+        /// not precisely match the size that the element is assigned.
+        public let contentMode: ContentMode
+
+        /// iOS 14 added support for Image Descriptions using VoiceOver. This is not always appropriate.
+        /// Set this to `true` to prevent VoiceOver from describing the displayed image.
+        public let blockAccessibilityDescription: Bool
+
+        public init(
+            tintColor: UIColor? = nil,
+            contentMode: ContentMode = .scaleToFill,
+            blockAccessibilityDescription: Bool = false
+        ) {
+            self.tintColor = tintColor
+            self.contentMode = contentMode
+            self.blockAccessibilityDescription = blockAccessibilityDescription
+        }
+    }
+
+    public convenience init(style: Style) {
+        self.init(frame: .zero)
+
+        translatesAutoresizingMaskIntoConstraints = false
+
+        tintColor = style.tintColor
+        contentMode = style.contentMode
+
+        if style.blockAccessibilityDescription {
+            // Seting `isAccessibilityElement = false` isn't enough here, VoiceOver is very aggressive in finding images to discribe.
+            // We need to explicitly remove the `.image` trait.
+            accessibilityTraits = .none
+        }
+    }
+}
+
+// MARK: ContentConfigurableView
+
+extension ImageRow: ContentConfigurableView {
+    public typealias Content = ImageType?
+
+    public enum ImageType: Equatable {
+        case image(UIImage?)
+        case asset(String)
+        case data(Data)
+        case file(String)
+        case sfSymbols(String)
+
+        public var image: UIImage? {
+            switch self {
+            case .image(let image):
+                image
+
+            case .asset(let name):
+                .init(named: name)
+
+            case .data(let data):
+                .init(data: data)
+
+            case .file(let path):
+                .init(contentsOfFile: path)
+
+            case .sfSymbols(let name):
+                .init(systemName: name)
+            }
+        }
+    }
+
+    public func setContent(_ content: Content, animated _: Bool) {
+        image = content?.image
+    }
+}
+
+// MARK: BehaviorsConfigurableView
+
+extension ImageRow: BehaviorsConfigurableView {
+    public struct Behaviors<T: UIImageView> {
```

Comment:
Remove `T`.

Reply:
In theory, this closure is only used to configure `UIImageView` image, but considering the inheritability of `UIImageView`, users may inherit `UIImageView` and add some properties related to image loading.

At this time, the generic type `T` can help users read those related properties when setting the image, because the closure parameter is the `Row` itself and not just the `UIImageView`.

Reply:
Ok.